src/transaction/types.ts

```diff
@@ -27,10 +29,16 @@ export type _Input = {
   outputIndex: number,
 }
 
+export type _Asset = {assetName: Buffer, coins: BigInt}
```

Comment:
coins is imho not the luckiest name - tokens may or may not be "coins". The terminology used for tokens is "amount"

Reply:
Refactored